manta-accounting/src/wallet/signer.rs

```diff
@@ -539,18 +579,56 @@ where
         deny_unknown_fields
     )
 )]
+#[derive(derivative::Derivative)]
+#[derivative(
+    Debug(bound = r"
+    AccountTable<C>: Debug,
+        C::UtxoAccumulator: Debug,
+        C::AssetMap: Debug,
+        C::Checkpoint: Debug,
+        C::Rng: Debug
+    "),
+    Default(bound = r"
+    AccountTable<C>: Default,
+        C::UtxoAccumulator: Default,
+        C::AssetMap: Default,
+        C::Checkpoint: Default,
+        C::Rng: Default
+    "),
+    Eq(bound = r"
+    AccountTable<C>: Eq,
+        C::UtxoAccumulator: Eq,
+        C::AssetMap: Eq,
+        C::Checkpoint: Eq,
+        C::Rng: Eq
+    "),
+    Hash(bound = r"
+    AccountTable<C>: Hash,
+        C::UtxoAccumulator: Hash,
+        C::AssetMap: Hash,
+        C::Checkpoint: Hash,
+        C::Rng: Hash
+    "),
+    PartialEq(bound = r"
+    AccountTable<C>: PartialEq,
+        C::UtxoAccumulator: PartialEq,
+        C::AssetMap: PartialEq,
+        C::Checkpoint: PartialEq,
+        C::Rng: PartialEq
+    ")
+)]
 pub struct SignerState<C>
 where
     C: Configuration,
 {
     /// Account Table
     ///
-    /// # Note
+    /// # Implementation Note
     ///
     /// For now, we only use the default account, and the rest of the storage data is related to
     /// this account. Eventually, we want to have a global `utxo_accumulator` for all accounts and
     /// a local `assets` map for each account.
-    accounts: AccountTable<C>,
+    pub accounts: AccountTable<C>, // This has been made public because of compatibility issues with sdk.
```

Comment:
Double check this. Do they need `&mut AccountTable<C>` access or is `&AccountTable<C>` enough?